manifests/e/ElcomsoftCo/ElcomsoftPhoneBreaker/10.12.38835.6971/ElcomsoftCo.ElcomsoftPhoneBreaker.locale.en-US.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ElcomsoftCo.ElcomsoftPhoneBreaker
+PackageVersion: 10.12.38835.6971
+PackageLocale: en-US
+Publisher: Elcomsoft Co. Ltd.
+PackageName: Elcomsoft Phone Breaker
+License: Trialwatr
+ShortDescription: Break Phones

```

Comment:
Can you provide a better description?

Reply:
Resolving this as I've suggested a paragraph from a potential [PackageUrl](https://www.elcomsoft.com/eppb.html) below.